src/DotNetWorker.Grpc/Factory/Handlers/FunctionEnvironmentReloadRequestHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
```

Comment:
Could you please add the copyright header?

Reply:
I assume I should do this for all files I have added, and not just this one?

Reply:
All new files :) You can copy it from any existing files in the repo.